bats/docs.bats

```diff
@@ -542,9 +544,6 @@ teardown() {
     run dolt table cp dolt_docs another_table
     [ "$status" -eq 1 ]
     [[ "$output" =~ "'dolt_docs' is not a valid table name" ]] || false
-    run dolt table create -s=`batshelper 1pk5col-ints.schema` dolt_docs
```

Comment:
I think the spirit of this test is to try and create a table named dolt_docs using a CREATE TABLE SQL statement

Reply:
`@test "dolt sql does not expose dolt_docs"` already has a check for `CREATE TABLE` in SQL